From e01843fb0ab14a09e70fd6df74da2800f08fcf21 Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Thu, 3 May 2007 09:22:24 +0000
Subject: [PATCH] parisc/clone.S: fix build failure on hppa.

gcc -I. -isystem include -pipe -nostdinc -Os -fomit-frame-pointer -Wall -W -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wno-switch -Wno-unused -Wredundant-decls -c parisc/clone.S -o bin-parisc/clone.o
parisc/clone.S: Assembler messages:
parisc/clone.S:40: Error: Invalid operands
make[1]: *** [bin-parisc/clone.o] Error 1
make[1]: Leaving directory `/home/pape/dietlibc'
make: *** [build-stamp] Error 2
---
 parisc/clone.S |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/parisc/clone.S b/parisc/clone.S
index 2846f3b..b504673 100644
--- a/parisc/clone.S
+++ b/parisc/clone.S
@@ -37,4 +37,4 @@ __clone:
 	bl	_exit,%r2		/* exit thread */
 	copy	%r28,%r26		/* return -> arg0 */
 
-	iitlbp	%r0,(%r0)		/* DIE ! DIE ! */
+	iitlbp	%r0,(%sr0,%r0)		/* DIE ! DIE ! */
-- 
1.6.0.3

